### PR TITLE
Refactor design system btns

### PR DESCRIPTION
**Pull request by bhushankhope**

### CODE REVIEW

File: libs/design-system/button/src/lib/button-styles/button-styles.component.scss [hunk 1]
```diff
@@ -41,14 +41,38 @@ $y-multiplier: 0.4566;
   }
 }
 
+// Custom Class for basic primary and secondary
+:is(a, button)[mat-button] {
+  &.button-size-large {
+    --mat-text-button-horizontal-padding: 1rem;
+    --mdc-text-button-container-height: 2.5rem;
+    font: var(--sys-label-large);
```
Comment (axdanbol) [inline libs/design-system/button/src/lib/button-styles/button-styles.component.scss:49]: Please also set letter-spacing whenever you set font, i.e. `letter-spacing: var(--sys-label-large-tracking);`

**Review by axdanbol — CHANGES_REQUESTED** [review]
Missing letter-spacing. Please ensure every font declaration has corresponding letter-spacing

**Comment by bhushankhope** [comment]
@axdanbol Libby has recently updated some styles of the buttons to match material 3 designs. I will also update these styles and then re request the review.

**Review by LibbyUX — APPROVED** [review]
Thank you so much @bhushankhope ! They look excellent!

**Review by axdanbol — APPROVED** [review]

**Pull request merged**